### PR TITLE
feat(display tokens used and total cost)

### PR DESCRIPTION
**Pull request by OpeOginni**
- Obtain tokens (input, output, total) for a test of a model
- Using Price calculate total cost of running model
- Display all value to user (can be selected to keep ones that are required)

### CODE REVIEW

File: app/play/[level]/test-mode.tsx [hunk 1]
```diff
@@ -28,6 +28,17 @@ export default function TestMode({ level, map }: TestModeProps) {
...
+  const [aiPromptTokensUsed, setAiPromptTokensUsed] = useState<number | null>(
+    null,
+  );
+  const [aiOutputTokensUsed, setAiOutputTokensUsed] = useState<number | null>(
+    null,
+  );
+  const [aiTotalTokensUsed, setAiTotalTokensUsed] = useState<number | null>(
+    null,
+  );
+  const [aiTotalRunCost, setAiTotalRunCost] = useState<number | null>(null);
```
Comment (delasy) [inline app/play/[level]/test-mode.tsx:40]: would be nice to have this info appear on playground page also, that means you will have to move this into custom hook

Comment (OpeOginni) [reply app/play/[level]/test-mode.tsx:40]: Alright.

I do have a question tho, on the playground page Models are fetched from the DB (doesnt work currently), while on the TestAI sections its gotten from a Constants file (this works). So which should be used?

**Comment by OpeOginni** [comment]
@delasy Is this sufficient?

**Comment by delasy** [comment]
@OpeOginni yep, need Cody to check it also

**Pull request merged**